index.js

```diff
@@ -26,6 +30,32 @@ var new_client = function(api_key, config) {
     throw new Error("You must configure the client with an API key");
   }
 
+  if (client.stream) {
+    es = new EventSource(config.stream_uri + '/features', {headers: {'Authorization': 'api_key ' + client.api_key}});
+    client.features = {};
```

Comment:
I think we should abstract this out into a `Store` interface, which can be backed with a persistent store like Redis. We'd probably want to throw a `version` field into the resource so that we can do optimistic concurrency?